lib/chef/mixin/versioned_api.rb

```diff
@@ -50,8 +50,7 @@ def get_class_for(type)
           # in which case we'll just start with the highest version and see what happens
           ServerAPIVersions.instance.min_server_version.nil? || (version >= ServerAPIVersions.instance.min_server_version && version <= ServerAPIVersions.instance.send(type))
         end
-          .sort { |a, b| a.send(:minimum_api_version) <=> b.send(:minimum_api_version) }
-          .last
+          .max { |a, b| a.send(:minimum_api_version) <=> b.send(:minimum_api_version) }
```

Comment:
```suggestion
          .max_by { |a| a.send(:minimum_api_version) }
```

Assuming the `send` is necessary, otherwise `.max_by(&:minimum_api_version)` would work.